Documentation/kubernetes-on-aws-render.md

```diff
@@ -161,6 +161,19 @@ You can now customize your cluster by editing asset files. Any changes to these
 [insecure-registry]: https://coreos.com/os/docs/latest/registry-authentication.html#using-a-registry-without-ssl-configured
 [update]: https://coreos.com/os/docs/latest/cloud-config.html#update
 
+### Kubernetes Container Runtime
+
+The kube-aws tool now optionally supports using rkt as the kubernetes container runtime. To configure rkt as the container runtime you must run CoreOS on the alpha channel and configure the runtime flag.
+
+Edit the `cluster.yaml` file:
+
+```yaml
+containerRuntime: rkt
+releaseChannel: alpha
+```
+
+Note that while using rkt as the runtime is now supported, it is still a new option as of the Kubernetes v1.3 release and may not yet be fully feature compliant. 
```

Comment:
"and has a few [known issues]\(http://kubernetes.io/docs/getting-started-guides/rkt/notes/\)." maybe? (not blocking on docs though, we can get this in a followup)